app/public/stylesheets/vendor/jquery.growl.css

```diff
@@ -0,0 +1,85 @@
+/* jQuery Growl
```

Comment:
Same here. You'll be setting the complete `jquery.growl` folder as public so requiring it from the html should be enough.

Reply:
😮 😮 😮

Reply:
And this